nncf/experimental/torch/fx/transformations.py

```diff
@@ -636,7 +880,7 @@ def separate_conv_and_bias(model: torch.fx.GraphModule):
     for n in model.graph.nodes:
         if not _is_conv(n):
             continue
-        if len(n.args) < 3 or n.args[2] is None:
+        if len(n.args) < 3 or n.args[2] is None or n.args[1].op != "get_attr":
```

Comment:
What's the purpose of this check?

Reply:
This is to ensure that the weight node is a constant. It was giving error in the case of a model with already inserted quantize-dequantize node for example:
weight   bias activation
    |           |        |
quantize  |        |   
    |           |        |
  add        |        |
    |           \|       |
dequantize \     |
     \             |    / 
         Conv2d
         
If the following pattern is passed, the convolution node does not have any check to ignore such a pattern.

Reply:
Please add a comment with description. From my understanding, it's not a problem of the approach but rather a not supported case.
Second question: where does this occur? I believe that quantized models should not be quantized twice

Reply:
It is an edge cases as @alexsu52 suggested. So, this minor change could help support such a case. I will add a comment